metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -213,6 +221,36 @@ def extract_lineage(
 
         return mcps
 
+    def create_datahub_owner_urn(self, user: str) -> str:
+        """
+        Create corpuser urn from PowerBI (configured by| modified by| created by) user
+        """
+        if self.__config.ownership.remove_email_suffix:
+            return builder.make_user_urn(user.split("@")[0])
+        return builder.make_user_urn(f"users.{user}")
+
+    def get_dataset_table_schema(
+        self,
+        field: Union[powerbi_data_classes.Column, powerbi_data_classes.Measure],
+    ) -> SchemaFieldClass:
+        data_type = field.dataType
+        if getattr(field, "expression", None):
```

Comment:
We could do `isinstance` check here instead of attribute check. for example isinstance of Column then determine the description from expression and description fields